hasura.planx.uk/migrations/1720597665798_create_table_s3_applications/up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE "public"."s3_applications" (
```

Comment:
My understanding is that these applications will be automatically picked up by the `submission_services_log` view as that's reading Hasura metadata tables directly right?

However, I think that the `submission_services_summary` view will require an update to also read from this new table.

Reply:
Good question - you're right: 
- `submission_services_log` is automatically picking up these records because it reads from Hasura events
- `submission_services_summary` will need to be manually updated (then Metabase too?) to join to this new audit table so analytics can be correctly tracked

I'd prefer to pick this up in a follow-up PR !

Reply:
Yep, makes total sense! ✅